src/scripts/main.js

```diff
@@ -1,3 +1,42 @@
 'use strict';
 
-// write code here
+const menu = document.querySelector('thead');
+const body = document.querySelector('tbody');
+const list = body.querySelectorAll('tr');
+
+menu.addEventListener('click', e => {
+  const item = e.target.closest('th');
+  const index = e.target.cellIndex;
+
+  if (!item || !menu.contains(item)) {
+    return;
+  }
+
+  const sortedCells = [...list]
+    .sort((a, b) => {
+      const aText = a.children[index].innerText;
+      const bText = b.children[index].innerText;
+
+      switch (index) {
+        case 0:
```

Comment:
Would be better to sort by labels, not index
if you have column reordering it's hard to fix this function for sorting